### DISCUSSION TITLE
Language leaders Discord 😎

### DISCUSSION THREAD

**Discussion by frenck** [OHF-Voice/intents#2270] [Announcements]
Oh hi, all of you language leaders 👋 

I wanted to let you know that we've opened up a Discord channel for our language leaders 🤘 
I've given access to a bunch of you I know are on Discord, but from a lot of you, I'm not sure who you are 😓 

Can you reach out to me (@`frenck` on Discord) or Missy (@`adorablecyborg` on Discord) so we can assign you the right role on the Home Assistant Discord chat? Or drop your Discord username here, that works too... Awesome 👍 

Also, If you are not on Discord yet... You should 😄 🎉 

<https://www.home-assistant.io/join-chat>

See you on the flip side 😁 

../Frenck

**Comment by frenck** [comment]
Oops, forgot to tag @home-assistant/language-leaders...

now I did 😀 

../Frenck

**Reply by larsdunemark** [reply]
coper1111 @ discord

**Reply by frenck** [reply]
Thanks, @larsdunemark! Added you 👍

**Reply by andrejs2** [reply]
andrejs2 @ discord 
Thanks @frenck

**Reply by frenck** [reply]
Thanks, @andrejs2! Added!

**Reply by witold-gren** [reply]
witoldg. @ discord

**Comment by missyquarry** [comment]
@home-assistant/language-leaders Hey, guys! Instead of creating a new discussion thread, I am gonna drag you all back into here since this will be Discord-related. First, let me say thanks to everyone for your continued work here, we really could not do this without you!!

I am working to get you guys the Voice Preview Edition and need y'all to fill out a form for me with your address. If you could please hit me up on Discord or leave a reply here with your Discord username so I can share the form with you directly, I would greatly appreciate it!! 🙏🏻 

We've only got about half of you on Discord with the Language Leader role - y'all have a dedicated private channel for your expertise in language intents, and we'll have a working group channel specifically for voice kit discussions. :) Please hit me up if you have any questions! :3

**Reply by al-one** [reply]
an.long

**Reply by luyzfernando08** [reply]
luyzfernando

**Reply by andrejs2** [reply]
andrejs2
👋

**Reply by bluefoxlee** [reply]
jerryleetw
Thanks.

**Reply by witold-gren** [reply]
witoldg.

**Reply by Uriziel01** [reply]
uriziel01.

**Reply by hertzg** [reply]
hertzg

**Reply by missyquarry** [reply]
Thanks, all! I got everyone above this comment in the #language-leaders channel in the Discord server and added the Developer role to those who didn't have it already so you can see the whole developer category.

Except you, @bluefoxlee. I am unable to find your username in the server. This happens sometimes so if you're already in there you can send me a DM, but if you aren't Frenck provided the link to the server in his OP of this thread. :D

If anyone else wants to drop their username here who haven't, please do! I'll comment to verify once I have added you. 😌

**Reply by bluefoxlee** [reply]
@missyquarry I just join the channel and DM you on Discord. Thanks.

**Comment by swonge** [comment]
Swonge
________________________________

**Reply by missyquarry** [reply]
I am not finding your username in the server. [Like I said here](https://github.com/home-assistant/intents/discussions/2270#discussioncomment-10753591), this sometimes happens. If you're in the server, you can DM me; but if not, use the link Frenck shared in the OP of this thread and lemme know when you are in. :)

**Reply by dontinelli** [reply]
> I am not finding your username in the server. [Like I said here](https://github.com/home-assistant/intents/discussions/2270#discussioncomment-10753591), this sometimes happens. If you're in the server, you can DM me; but if not, use the link Frenck shared in the OP of this thread and lemme know when you are in. :)

dontinelli

**Reply by missyquarry** [reply]
@dontinelli Thanks, got you added! :D

**Comment by mojikosu** [comment]
mojikosu on discord

Thank you.

**Reply by missyquarry** [reply]
Gotchu added into the channel, thank you!!

**Comment by LubosKadasi** [comment]
LubosKadasi

**Reply by missyquarry** [reply]
You are all set, thank you!

**Comment by spuljko** [comment]
Gond

**Reply by missyquarry** [reply]
Got you added! You'll get a DM from me shortly. 😄

**Comment by missyquarry** [comment]
@home-assistant/language-leaders Final call for this comment, y'all! https://github.com/home-assistant/intents/discussions/2270#discussioncomment-10743371

We're well on the way to getting these out to you all and I am missing a few people still (based on total responses on the survey I shared out). Lemme know if you still aren't on the server, need access to the Language Leaders channel, and if you need the link for the survey to give us your address. :D

**Reply by v1k70rk4** [reply]
Hi, I was out for three months due to illness, but I’m back now :) Discord #v1k70rk4

**Reply by Arkkimaagi** [reply]
Arkkis#0470

**Reply by makstech** [reply]
real_ovich

**Reply by missyquarry** [reply]
Thanks, guys! I got you all added to the channel on the server and you should have a private message from me with the survey link for you to fill out. 😄

**Reply by ernst77** [reply]
ernest_st

**Reply by Natfaji** [reply]
Natfaji_A

**Comment by arunshekher** [comment]
`skepticautonetic`
